src/sdk/config_fs.js

```diff
@@ -103,13 +103,13 @@ class ConfigFS {
 
     /**
      * add_config_file_suffix returns the config_file_name follwed by the given suffix
+     * Bucket name can have suffix `.json`. This will make sure the bucket name and bucket config file have the same name.
      * @param {string} config_file_name
      * @param {string} suffix
      * @returns {string} 
      */
     add_config_file_suffix(config_file_name, suffix) {
         if (!config_file_name) dbg.warn(`Config file name is missing - ${config_file_name}`);
```

Comment:
do we have the same/similar issue with symlink? like account name that ends with symlink?

Reply:
yes, same issue was there for `.symlink` suffix account names too, but with this change it got fixed.